### PR TITLE
Add 'finish-args-unnecessary-xdg-data-access' for net.nokyan.Resources

### PR DESCRIPTION
**Pull request by nokyan**
Resources is a process viewer and system monitor that also detects and displays running applications.
In order to find user-wide installed Flatpak applications, Resources needs access to xdg-data.

### CODE REVIEW

File: flatpak_builder_lint/staticfiles/exceptions.json [hunk 1]
```diff
@@ -1680,7 +1680,8 @@
         "finish-args-flatpak-spawn-access": "Required to get information about the os and execute host commands e.g. like the package managers"
     },
     "net.nokyan.Resources": {
-        "finish-args-flatpak-spawn-access": "Required to spawn a small executable dedicated to finding running processes in /proc"
+        "finish-args-flatpak-spawn-access": "Required to spawn a small executable dedicated to finding running processes in /proc",
+        "finish-args-unnecessary-xdg-data-access": "Required to detect user-wide installed Flatpak applications by searching for .desktop files"
```
[the remark this answers is not in the document]
Comment (bbhtt) [reply flatpak_builder_lint/staticfiles/exceptions.json:1684]: The app already has spawn access.

**Comment by bbhtt** [comment]
LGTM, it correctly checks the path relative to XDG_DATA_DIRS first https://github.com/nokyan/resources/blob/a637a1e53bbea1d0e754c004ee052680844d0964/src/utils/app.rs#L33

File: flatpak_builder_lint/staticfiles/exceptions.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (bbhtt) [reply flatpak_builder_lint/staticfiles/exceptions.json:1684]: https://github.com/flatpak/xdg-desktop-portal/issues/653 might help here, although I didn't look closely at how noykan uses this.

[the remark this answers is not in the document]
Comment (nokyan) [reply flatpak_builder_lint/staticfiles/exceptions.json:1684]: I could use spawn to get the installed applications, I just guessed that it'd be more appropriate to not use spawn when something else is possible 🤷🏻

**Pull request merged**